### PR TITLE
Data preprocessing

### LINKED ISSUE #2: Preprocess data

### PR DESCRIPTION
**Pull request by christianjans**
Will close #2.

### CODE REVIEW

File: src/data/stopwords.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
+"""
+Obtained from NLTK's list of english stopwords
+
+Functional requirements: FR8
```
Comment (jboileau99) [inline src/data/stopwords.py:4]: is this related to our SRS?

**Review by jboileau99 — APPROVED** [review]
Looks great Chris!

File: src/data/stopwords.py (hunk 1 above)
Comment (christianjans) [reply src/data/stopwords.py:4]: Oh sorry, thanks for catching it. No it's not, these are the stopwords from the other group's project. I think they're relevant to include, but not part of our FRs.
